src/app/shared/mastodon/mastodon.component.html

```diff
@@ -0,0 +1,103 @@
+<!--
```

Comment:
It's not clear to me whether or not this HTML was adopted, could you add a comment clarifying if it was/wasn't?

Reply:
added a comment